ores/score_request.py

```diff
@@ -73,3 +73,27 @@ def __repr__(self):
                 "injection_caches={0!r}".format(self.injection_caches),
                 "ip={0!r}".format(self.ip),
                 "model_info={0!r}".format(self.model_info)]))
+
+    def toJSON(self):
```

Comment:
I don't think "JSON" is the right word for what we're doing here.  Technically, it's "marshalling" although that doesn't exactly roll off the tongue.  Also, Python breaks the term by misusing it as another name for serialization.

What do you think about "toData" and "newFromData"?

Reply:
Can you say more why you don't think "JSON" is the right term?  Maybe you mean "JSONable" since it's not turning the data structure into JSON, but rather a trivially JSON-able python data structure.

Reply:
Oh also the name is a style issue.  Should probably be "to_json" or something like that.

Reply:
Renamed them to to_json and from_json

Reply:
I renamed it to to_json